src/components/SendButton/sendButton.module.css

```diff
@@ -0,0 +1,21 @@
+.btn {
+  background-color: black;
+  border-radius: 50px 50px 0 0;
```

Comment:
I guess this is fine for the moment as we're not worrying about responsiveness, but 50px will be pretty big on mobile.